docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -39,8 +39,9 @@ Run the following from PowerShell to deploy all the prerequisites for building O
 ./scripts/install-windows-prereqs.ps1
 ```
 
-On Windows 10, the Intel PSW is delivered via Windows Update. Running the
-executable installer will fail on Windows 10 machines. To skip PSW installation:
+On Windows 10, the Intel PSW is delivered via Windows Update, and on Windows Server 2019, the Intel
```

Comment:
I believe both Windows 10 and Window Server 2019 behaves the same (i.e., auto-included via Windows Update).